srcpkgs/gcc/template

```diff
@@ -364,6 +363,8 @@ do_install() {
 	ln -sfr ${DESTDIR}/usr/include/c++/${_minorver} \
 		${DESTDIR}/usr/include/c++/${version}
 
+	# cc symlink
+	ln -sfr ${DESTDIR}/usr/bin/gcc ${DESTDIR}/usr/bin/cc
```

Comment:
target should be `/usr/bin/gcc` or `gcc`.

Reply:
Please clarify, I don't see the issue.

Reply:
Sorry, i missed -r flag. It's fine.